homeassistant/components/saj/__init__.py

```diff
@@ -1 +1,78 @@
 """The saj component."""
+
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+import pysaj
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    CONF_HOST,
+    CONF_PASSWORD,
+    CONF_TYPE,
+    CONF_USERNAME,
+    Platform,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+
+from .const import CONNECTION_TYPES
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
```

Comment:
You should extend the configEntry type to make your own type, and put the type of entry.runtime_data into the typslot (check the `runtime-data` integration quality scale rule for explanation)